include/openmc/particle_data.h

```diff
@@ -227,9 +228,11 @@ class ParticleData {
   int g_last_;    //!< pre-collision energy group (MG only)
 
   // Other physical data
-  double wgt_ {1.0};  //!< particle weight
-  double mu_;         //!< angle of scatter
-  bool alive_ {true}; //!< is particle alive?
+  double wgt_ {1.0};       //!< particle weight
+  double mu_;              //!< angle of scatter
+  double time_ {0.0};      //!< time in [s]
+  double time_last_ {0.0}; //!< previous time in [s]
```

Comment:
A more general comment (probably due to my lack of knowledge about certain structures): Why do we take the detour via time_last and time? Can't we just do

time = time + distance / speed.

There is a similar structure when it comes to E and E_last_ which I used for the PHT. Maybe soon we could use it to access the times between interactions, or what is the idea here?

Reply:
For tallies that occur after a particle collision, the speed of the particle has already changed and hence you can't backtrack like this to figure out what time the particle started its track at.

Reply:
thank you